modules/adnuntiusBidAdapter.js

```diff
@@ -146,50 +119,66 @@ export const spec = {
 
   interpretResponse: function (serverResponse, bidRequest) {
     const adUnits = serverResponse.body.adUnits;
+
+    function buildAdResponse(ad, adUnit, isDeal, dealCount) {
+      let destinationUrls = ad.destinationUrls || {};
+      let advertiserDomains = [];
+      for (const value of Object.values(destinationUrls)) {
+        advertiserDomains.push(value.split('/')[2])
+      }
+      let adResponse = {
+        bidderCode: isDeal ? BIDDER_DEAL_ALIAS + (dealCount || 1) : BIDDER_CODE,
```

Comment:
This isn't allowed, it breaks aliasing

Reply:
Can you clarify which part of this breaks aliasing? I have tested this code and it functionally "works", but perhaps it breaks things in a more subtle way that I don't understand. 

Is the issue that the request is made using the `adnuntius` bidder code, but the response can come back with both an `adnuntius` bidder code and also an `adndeal1` and `adndeal2` etc bidder code?

Reply:
as you can see in #10232, your changes broke aliasing. Publishers should be able to choose any bidder code they want. Apologies for just getting back to this, but when it was marked as draft it fell completely off my radar.